pkg/catalog/helpers_test.go

```diff
@@ -125,6 +129,8 @@ func newFakeMeshCatalogForRoutes(t *testing.T) *MeshCatalog {
 	mockKubeController.EXPECT().IsMonitoredNamespace(tests.BookbuyerService.Namespace).Return(true).AnyTimes()
 	mockKubeController.EXPECT().ListMonitoredNamespaces().Return(listExpectedNs, nil).AnyTimes()
 
+	mockConfigurator.EXPECT().IsPermissiveTrafficPolicyMode().Return(testParams.permissiveMode).AnyTimes()
```

Comment:
Note that `AnyTimes()` is greedy, so you can't change the expectations for this mock call once `AnyTimes()` has been set.

This is another reason I suggest not initializing mocks in these fake functions but rather directly in the test.